packages/components/pages/f-loyalty/src/components/HowItWorks.vue

```diff
@@ -152,44 +143,42 @@ export default {
             // _Very_ naive way of implementing ordinals but will suit our purposes here
             const ordinal = { 0: 'st', 1: 'nd', 2: 'rd' };
 
-            return this.bagsData.map((bagValue, index) => ({
+            return [...this.$t('howItWorks.bagValues')].map((bagValue, index) => ({
                 number: `${index + 1}${ordinal[index] || 'th'}`,
```

Comment:
What is this number used for? If it's just a position, shouldn't be just an int/short? If it's used anywhere, any potential issues for other languages other than English?

Reply:
stated in another comment